### PR TITLE
Pf4 updates

### PR DESCRIPTION
**Pull request by dlabaj**
## Motivation
Added PF4 to mobile developer console and updated the Empty State Component

## Verification Steps
Verified that the Empty state was updated and the tests passed.

Screen Shot 2019-07-23 at 3 37 14 PM

## Checklist:

- [x] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Progress

- [x] Finished task

### CODE REVIEW

File: src/components/overview/MobileClientCardView.js [hunk 1]
```diff
@@ -11,7 +13,8 @@ class MobileClientCardView extends Component {
     this.state = { filter: '', currentValue: '' };
 
     this.emptyStateMessage = {
-      noAppsCreated: 'You have no mobile apps right now. Create one to get started.',
+      noAppsCreatedTitle: "You don't have any Mobile Apps.",
```
Comment (craicoverflow) [inline src/components/overview/MobileClientCardView.js]: Our [eslint configuration](https://github.com/aerogear/mobile-developer-console/blob/c151bb4532b63358224f41bee47a4ef78aa55850/.eslintrc.json) is set to prefer single quotes for JS files. I'm not sure why, but eslint is not actually picking up that specific line for some reason.

See below, I changed the next property in the object which gives an error, but not in `noAppsCreatedTitle`.

Screenshot from 2019-08-01 10-08-34

File: package.json [hunk 2]
```diff
@@ -3,6 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@patternfly/patternfly": "^2.23.0",
```
Comment (craicoverflow) [inline package.json]: Is it better to fix the version of this?

```suggestion
"@patternfly/patternfly": "2.23.0"
```

File: package.json [hunk 3]
```diff
@@ -3,6 +3,9 @@
...
+    "@patternfly/react-core": "^3.58.1",
```
Comment (craicoverflow) [inline package.json]: Is it better to fix the version of this?

```suggestion
"@patternfly/react-core": "3.58.1"
```

File: package.json [hunk 4]
```diff
@@ -3,6 +3,9 @@
...
+    "@patternfly/react-icons": "^3.10.13",
```
Comment (craicoverflow) [inline package.json]: Is it better to fix the version of this?

```suggestion
"@patternfly/react-icons": "3.10.13"
```

File: src/components/overview/MobileClientCardView.js (hunk 1 above)
Comment (dlabaj) [reply src/components/overview/MobileClientCardView.js]: I'll update these to single quotes and fix the button.

File: package.json (hunk 2 above)
Comment (dlabaj) [reply package.json]: yep all these should be fixed.  Will update these thanks

**Comment by dlabaj** [comment]
Canceling PR we are going to do another PR with PF4 and Masthead changes.

**Pull request closed without merging**